commerce-order-content-web/src/main/java/com/liferay/commerce/order/content/web/internal/display/context/CommerceOrderContentDisplayContext.java

```diff
@@ -493,7 +493,7 @@ public boolean isShowPurchaseOrderNumber() throws PortalException {
 	private final CommercePaymentMethodGroupRelService
 		_commercePaymentMethodGroupRelService;
 	private final CommerceShipmentItemService _commerceShipmentItemService;
-	private final CPRequestHelper _cpRequestHelper;
+	private final CommerceOrderRequestHelper _cpRequestHelper;
```

Comment:
`CommerceOrderRequestHelper _cpRequestHelper` should be `CommerceOrderRequestHelper _commerceOrderRequestHelper`